assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -545,6 +662,21 @@ const baseSelectors = {
 		// update.
 		return state.isAwaitingModulesRefresh;
 	} ),
+
+	canActivateModule( state, slug ) {
+		const moduleRequirements = state.checkRequirementsResults?.[slug];
+
+		if ( moduleRequirements === undefined ) {
+			return undefined;
+		}
+
+		return moduleRequirements === true;
+	},
+
+	getCheckRequirementsStatus( state, slug ) {
+		invariant( slug, 'slug is required.' );
+		return state.checkRequirementsResults[ slug ];
```

Comment:
This should only be returned if it is an error object. If it's not set, `undefined` should be returned, or if it's `true`, `null` should be returned (there is no error).

Maybe it would be better to name this selector `getCheckRequirementsError` to make its purpose more clear?

Reply:
Updated.